modules/nw-install-config-parameters.adoc

```diff
@@ -23,7 +23,7 @@ You cannot modify these parameters after installation.
 |`networking.networkType`
 |The network plug-in to deploy. The `OpenShiftSDN` plug-in is the only plug-in
 supported in {product-title} {product-version}. The `OVNKubernetes` plug-in is
-available as a Tech Preview in {product-title} 4.2.
+available as a Tech Preview in {product-title} {product-version}.
```

Comment:
But it will be GA in 4.3; Does it make sense to use the variable instead here?

Reply:
Just in case, I switched it back. Thanks!